.github/workflows/run-preview-index-test.yaml

```diff
@@ -0,0 +1,27 @@
+name: Run preview index test
+
+on:
+  workflow_dispatch: # Allow manual trigger
+  repository_dispatch:
+    types:
+      - resource-previewed
+
+jobs:
+  print:
+    runs-on: ubuntu-latest
+    steps:
+      - run: |
+          echo "Status: ${{ github.event.client_payload.status }}"
+          echo "Path: ${{ github.event.client_payload.path }}"
+  run-preview-index-test:
+    name: Test reindexing preview resources
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [18.x]
+
+    steps:
+      - name: Test reindex preview resources
+        run: |
+          cd .github/preview-index
+          npm run testevent --payload=${{ toJson(github.event.client_payload) }}
```

Comment:
i think it will fail because you are missing the 'Set up Node.js ${{ matrix.node-version }}' step

not sure if `npm install` is crucial but might be needed too.

Reply:
Added